Prepare for release v0.2.2 (#82)
DataDog/opentelemetry-mapping-go: CHANGELOG.md

<!-- next version -->

## v0.2.2

### 🛑 Breaking changes 🛑

> - `pkg/otlp/metrics`: Changes MapMetrics function to add a new return value `RuntimeMetricsTelemetry`, which provides runtime metrics information on a MapMetrics run. (#81)

Reviewer: ```suggestion
- `pkg/otlp/metrics`: the MapMetrics method now returns runtime metrics information as well. (#81)
```
The current phrase seems like going around in circles saying the same thing :)
